src/tasks/netcore/NetCoreTaskHelper.ts

```diff
@@ -280,7 +280,9 @@ export class NetCoreTaskHelper implements TaskHelper {
             addVolumeWithoutConflicts(volumes, srcVolume);
             addVolumeWithoutConflicts(volumes, debuggerVolume);
             addVolumeWithoutConflicts(volumes, nugetVolume);
-            addVolumeWithoutConflicts(volumes, nugetFallbackVolume);
+            if (fse.pathExistsSync(nugetFallbackVolume.localPath)) {

```

Comment:
Any reason not to use the "async" version?

Reply:
No reason. Will use async version